doc/install.txt

```diff
@@ -8,43 +8,25 @@ Installing Theano
 Requirements
 ------------
 
-In order to use Theano, the following libraries and software will need
-to be installed (MacOS and Windows users should refer to platform-specific
-instructions below for detailed installation steps):
+Theano is mainly develeopped on 64-bit Linux machines. For other architectures,
+please refer to :ref:`install_others`
```

Comment:
Let's keep the prerequisites list very minimal and move the "extra text" (e.g. "the CentOS installation It covers how to install Theano (for CPU-based computation only) with the distribution-packaged ATLAS, a free fast implementation of BLAS.") inside the OS-specific installation pages (e.g. in the CentOS installation page).